data/reusables/organizations/review-team-repository-access.md

```diff
@@ -1 +1,3 @@
-1. Review the list of repositories that the new team member will have access to, then click **Add _USERNAME_ to _TEAMNAME_**.
+1. Review the list of repositories that the new team member will have access to, then click **Add USERNAME to TEAMNAME**.
+  ![Modal box with a list of the repositories the new team member will have access to and confirmation button](/assets/images/help/teams/add-team-member-repo-perms.png)
+
```

Comment:
💡 Turns out we removed this image [on `main`](https://github.com/github/docs/blob/main/data/reusables/organizations/review-team-repository-access.md?plain=1). We need to remove this Markdown to get the checks passing.

```suggestion
```